src/jsonrpc/mod.rs

```diff
@@ -26,7 +31,7 @@ pub const NO_PARAMS: Value = json!([]);
 #[async_trait]
 pub trait JsonRpcClient {
```

Comment:
You're making this change. You need to assume ownership that the code it affects remains cohesive. It's totally understandable that you don't want to make this change as part of this PR, but you should then at least add a TODO comment and make sure that you or someone else follow up on it. It might require opening an issue in addition to the TODO.

Reply:
I will add the TODO